### ISSUE TITLE
Got manifest.[hash].json but dont know how to inject it on custom template

### ISSUE THREAD

**Issue by andresmatasuarez** [arthurbergmz/webpack-pwa-manifest#66]
Hi! First of all, I want to thank you guys for this awesome plugin. 

Now, what is happening to me is this: I need to add a web app manifest.json file to my web app to make it Progressive. Im indeed getting the file generated by the plugin, but since Im not using HtmlWebpackPlugin, Im not being able to include/inject it.

For my project, I rely on `assets-webpack-plugin` to get a `webpack-assets.json` with all the mappings of entry name vs hashed file, and in my custom template (.pug file, ex .jade) I include a reference to them. That's how I get all my js/css outputs working.

So my question is this: is there a way to add the manifest link tag to my custom template? 

In other words, after each webpack build I end up with a manifest.[hash].json file that I cannot reference by name in my html/pug since the hash always changes with each build.

**Comment by Onset** [comment]
I believe it is related to https://github.com/arthurbergmz/webpack-pwa-manifest/issues/32. Webmanifest file is not recognized as other assets.
